manifests/grafana-dashboardDatasources.yaml

```diff
@@ -19,7 +19,7 @@ stringData:
                 "name": "prometheus",
                 "orgId": 1,
                 "type": "prometheus",
-                "url": "http://prometheus-k8s.monitoring.svc:9090",
+                "url": "http://prometheus-operated.monitoring.svc:9090",
```

Comment:
The service name is based on name defined for prometheus service in manifest https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/prometheus-service.yaml#L10